docs/notebooks/verification_gridded_cosmo.ipynb

```diff
@@ -0,0 +1,3097 @@
+    "- The ML-model is called Neural-LAM and is initialised with the deterministic analysis.\n",
+    "- The Ground Truth is the same deterministic analysis as was used to train the ML-model.\n",
+    "- The boundary data for both models is IFS HRES from ECMWF, where the NWP-model got 6 hourly boundary updates and the ML model 12 hourly.\n",
+    "\n",
+    "For more info about the COSMO model see:\n",
+    "- https://www.cosmo-model.org/content/model/cosmo/coreDocumentation/cosmo_io_guide_6.00.pdf\n",
+    "- https://www.research-collection.ethz.ch/handle/20.500.11850/720460"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c0311b50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import random\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import cartopy.crs as ccrs\n",
+    "import cartopy.feature as cfeature\n",
+    "import dask\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import xarray as xr\n",
+    "from dask.distributed import Client, LocalCluster\n",
+    "from pysteps.verification.salscores import sal  # requires scikit-image\n",
+    "from scipy.stats import wasserstein_distance, kurtosis, skew\n",
+    "from scores.continuous import (\n",
+    "    mae,\n",
+    "    mse,\n",
+    "    rmse,\n",
+    ")\n",
+    "from scores.continuous.correlation import pearsonr\n",
+    "from scores.spatial import fss_2d"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a806b47",
+   "metadata": {},
+   "source": [
+    "**--------> Enter all your user settings in the cell below. <--------**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cf621a4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### DEFAULTS ###\n",
+    "# This config will be applied to the data before any plotting. The data will be\n",
+    "# sliced and indexed according to the values in this config.\n",
+    "PATH_GROUND_TRUTH = \"/iopsstor/scratch/cscs/sadamov/pyprojects_data/neural-lam/cosmo.datastore.zarr\"\n",
+    "PATH_NWP = \"/capstor/store/cscs/swissai/a01/sadamov/cosmo_e_forecast.zarr\"\n",
+    "PATH_ML = \"/iopsstor/scratch/cscs/sadamov/pyprojects_data/neural-lam/eval_results/train-hi_lam-2x300-02_12_05-1462/test.zarr\"\n",
+    "PATH_BOUNDARY = \"/iopsstor/scratch/cscs/sadamov/pyprojects_data/neural-lam/era5_7_19_margin_interior.datastore.zarr\"\n",
+    "# elapsed forecast duration in steps for the forecast - [0] refers to the first forecast step\n",
+    "ELAPSED_FORECAST_DURATION = [0, 3, 6, 9, 12, 15, 18, 21]\n",
+    "# Select specific start_times for the forecast (will be used for all variables and metrics)\n",
+    "START_TIMES = [\"2020-02-13T00:00:00\", \"2020-02-15T00:00:00\"]\n",
+    "# Select specific plot times for the forecast (will be used to create maps for all variables)\n",
+    "PLOT_TIME = \"2020-02-13T00:00:00\"\n",
```

Comment:
If I want to make plots for multiple times, should I set this to a list, or do I just run the notebook (that specific cell, that is) multiple times?

Reply:
here you can only enter one time, and then if you want another you rerun that cell. because plotting the maps takes a very long time and I find it better to limit this to one specific start_time of interest.